engine/src/main/java/org/destinationsol/game/console/commands/InvincibleCommandHandler.java

```diff
@@ -48,13 +43,17 @@ public void handle(String input, Console console) {
             console.warn("Cannot make invincible when transdencent");
             return;
         }
-        if (!isInvincible) {
-            console.warn("Set player as invincible");
-            SolShip.setInvincible(true);
-        } else if (isInvincible) {
-            console.warn("Set player as non invincible");
-            SolShip.setInvincible(false);
+        if (!hero.isInvincible()) {
+            console.info("Set player as invincible");
+            hero.setInvincible(true);
+        } else if (hero.isInvincible()) {
+            console.info("Set player as non invincible");
+            hero.setInvincible(false);
         }
     }
 
+    public boolean isInvincible() {
```

Comment:
Why is this method even here? Like, ideally, nothing should ever have an instance of this class apart from console itself, which does not know what to do with it in place.